app/code/Magento/BundleGraphQl/etc/schema.graphqls

```diff
@@ -1,5 +1,5 @@
-# Copyright © Magento, Inc. All rights reserved.
-# See COPYING.txt for license details.
+# Copyright 2018 Adobe.
+# All Rights Reserved.
```

Comment:
is this correct? it doesn't seem like an open source header and I see the old one in the rest of the code.

Reply:
What's not correct about it? It follows the new copyright template Adobe's legal team now requires, like I already mentioned in another comment a few months ago to you:

```
Copyright [year code created] Adobe
All Rights Reserved.
```

Other recent commits do have problematic copyright headers, and I already escalated it to internal Adobe devs a few days ago:
<img width="1214" alt="Screenshot 2025-02-19 at 22 17 56" src="https://github.com/user-attachments/assets/1be360a1-4005-4449-b146-78f9a42d0909" />

Reply:
"all rights reserved" sounds like "full copyright" and not really like OSL-3. 
If there wasn't no repository-level license I'd assume this is not an open source project...

Reply:
It's incongruous, but it's what Adobe powers that be settled on, and the project as a whole (which this code is part of) is still licensed OSL/AFL regardless.